_episodes/21-carpentries.md

```diff
@@ -14,40 +15,53 @@ keypoints:
 
 ---
 
-In becoming a Carpentry instructor,
+In becoming a certified [Carpentry instructor](https://carpentries.org/instructors/),
```

Comment:
I believe this should be "Carpentries instructor" not "Carpentry instructor"

Reply:
correct!